app/views/bamposts/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing Bampost</h1>
+
+<%= render 'form', bampost: @bampost %>
+
+<%= link_to 'Show', @bampost %> |
+<%= link_to 'Back', bamposts_path %>
```

Comment:
Great job using appropriate partials, helper methods and replacing your instance variables with the local ones! 👍 ✅